src/main/java/frc/robot/Constants.java

```diff
@@ -82,6 +82,11 @@ private DriveConstants() {
         (WHEEL_DIAMETER_METERS * Math.PI) / GEAR_RATIO;
     public static final double ENCODER_VELOCITY_CONVERSION =
         (WHEEL_DIAMETER_METERS * Math.PI) / (GEAR_RATIO * 60);
+
+    // Starting field position for odometry
```

Comment:
Could you elaborate a little bit? If you hadn't showed me the field sim last night I wouldn't know what this is about.

Reply:
I will add some more explanation.  It is actually not just for simulation but also important for the real robot for trajectory following and for replaying drive tests from data logs.  With gyro drift and wheel slip the pose (position and heading) will accumulate errors over time.  If you want to test multiple runs with restarting the robot you need a defined start pose and a command to reset. For autonomous operations we may need a selector to pick from a set of possible starting positions.  You could just make every command start at 0,0, but then you lose the ability to display trajectories on the field or replay the match on the field.